javascript-timers/interval.js

```diff
@@ -0,0 +1,15 @@
+var $h1 = document.querySelector('h1');
+
+setInterval(changeTextContent, 1000);
+
+function changeTextContent() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+  } else if ($h1.textContent === '1') {
```

Comment:
This isn't very maintainable. If you ever wanted to start at a number like 100, you'd have to write a lot of conditional blocks. You should use a variable to hold the current number, and just decrement it each time the code runs